Marlin/Marlin_main.cpp

```diff
@@ -2182,6 +2182,11 @@ static void clean_up_after_endstop_or_probe_move() {
     #if ENABLED(BLTOUCH)
       if (set_bltouch_deployed(true)) return true;
     #endif
+	
+    #if ENABLED(WAIT_FOR_BED_HEATER)
+      // Wait for bed to heat back up between probing points
+      while (thermalManager.isHeatingBed()) safe_delay(200);
+    #endif
```

Comment:
My prior caveat still stands. This behavior will sometimes be obnoxious because it affects `G28`, `G29`, `G30`, `M48`, and every other command that uses the probe. It gives no feedback to the user indicating why the machine suddenly stops at the start of a probe, or in the middle of multiple probes.

Reply:
Actually it doesnt wait on homing. Home uses different code. But thanks for pointing that out i've added it to homing now too.

Reply:
What is the best way to notify the user? So far ive added a serial out but maybe they should be notified to the LCD?

Reply:
I think my echo's are being blocked by the movement:

3:12:26.938: Waiting for bed to heat...echo:busy: processing (2)
3:12:32.522: X:192.00 Y:141.00 Z:13.45 E:0.00 Count X:15360 Y:11280 Z:5380
3:12:47.665: echo:DEBUG:ECHO
3:12:52.012: echo:N14 G28*38
3:13:10.040: Waiting for bed to heat...echo:busy: processing

They seem to happen after the movement has finished